### PR TITLE
Create project initial with models

### PR DESCRIPTION
**Pull request by Linzeur**
Create Project
Create Structure Folder
Add namespace Api in Routes
Execute commands according to instructions in Extended Project

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.6.1'
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
+gem 'rails', '~> 5.2.3'
+# Use postgresql as the database for Active Record
+gem 'pg', '>= 0.18', '< 2.0'
+# Use Puma as the app server
+gem 'puma', '~> 3.11'
```
Comment (deyviconde) [inline Gemfile]: I feel curious about other alternatives instead of Puma 🐯

**Comment by CesarCachay** [comment]
Hey dude, I think you can create all the models (albums & artists) to start coding all the group :+1:

File: config/routes.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  namespace :api do
```
Comment (deyviconde) [inline config/routes.rb]: Cool, we are 'wrapping' everything inside api namespace

File: Gemfile (hunk 1 above)
Comment (Linzeur) [reply Gemfile]: Now not young!

File: spec/controllers/albums_controller_spec.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe Api::AlbumsController, type: :controller do
+
+  describe "GET #new" do
+    it "returns http success" do
+      get :index
```
Comment (KattyaCuevas) [inline spec/controllers/albums_controller_spec.rb]: The describe of this block is `GET #new`, why are you doing a `get :index` ?

Comment (Linzeur) [reply spec/controllers/albums_controller_spec.rb]: It was a mistake, it was already fixed

**Pull request merged**